compiler/quilt/tools/main.py

```diff
@@ -96,8 +97,12 @@ def check_hash(group, hashstr):
     tag_remove_p.set_defaults(func=command.tag_remove)
 
     install_p = subparsers.add_parser("install")
-    install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename",
-                           nargs="?", default="@"+DEFAULT_QUILT_YML)
+    if os.path.exists(DEFAULT_QUILT_YML):
+        install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename",
+                               nargs="?", default="@"+DEFAULT_QUILT_YML)
+    else:
+        install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename",
+                               nargs="+")
```

Comment:
rofl forehead-smacker :-)

install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename", nargs="?", default=("@"+DEFAULT_QUILT_YML) if os.path.exists(DEFAULT_QUILT_YML) else None)

Reply:
No problem with a ternary operation, but using that would require two ternary operations:
```
install_p.add_argument("package", type=str, help="owner/package_name[/path/...] or @filename",
       nargs="?" if os.path.exists(DEFAULT_QUILT_YML) else "+",
       default=("@"+DEFAULT_QUILT_YML) if os.path.exists(DEFAULT_QUILT_YML) else None)
```
...would that method still be preferred?

Reply:
The if/else seems easier to read to me, but I'm ok with either. Either way, please add a comment explaining the behavior.

Reply:
added comment.